client/flutter/lib/main.dart

```diff
@@ -87,14 +96,18 @@ class _MyAppState extends State<MyApp> {
         accentColor: Constants.textColor,
         brightness: Brightness.light,
         buttonTheme: ButtonThemeData(
-            buttonColor: Constants.primaryColor,
-            textTheme: ButtonTextTheme.accent),
+          buttonColor: Constants.primaryColor,
+          textTheme: ButtonTextTheme.accent,
+        ),
       ),
       home: Directionality(
-          child: HomePage(analytics),
-          textDirection:
-              GlobalWidgetsLocalizations(Locale(Intl.getCurrentLocale()))
-                  .textDirection),
+        child: widget.showOnboarding
+            ? OnboardingPage(analytics: analytics)
```

Comment:
```suggestion
            ? OnboardingPage(analytics)
```

Reply:
Why?

Reply:
I agree that named parameters are better in the longer term, but I also agree that while we're not refactoring for v1.0, we should keep our standards consistent. Since `HomePage` uses a positional parameter, let's keep that for now for the `OnboardingPage` too